### PR TITLE
add support link

### PR DESCRIPTION
**Pull request by thinkC**

### CODE REVIEW

File: roles/azimuth/defaults/main.yml [hunk 1]
```diff
@@ -85,6 +85,7 @@ azimuth_metrics_tenant_metrics_url_template: >-
 # The documentation URL
 # By default, we use the generic Azimuth docs
 azimuth_documentation_url: https://azimuth-cloud.github.io/azimuth-user-docs/
+azimuth_support_url: https://confluence.skatelescope.org/display/SRCSC/Azimuth+Support+on+Cambridge+Arcus
```
Comment (mkjpryor) [inline roles/azimuth/defaults/main.yml:88]: Similar to the Azimuth PR, this should not have a value by default.

**Review by mkjpryor — CHANGES_REQUESTED** [review]

**Comment by thinkC** [comment]
Hello @mkjpryor, I changes made . Kindly review

**Review by JohnGarbutt — COMMENTED** [review]
This looks worth adding.

We might also want to tweak to the docs so it's clear this is an option? Maybe that is overkill. I will let @mkjpryor re-review this one.

File: roles/azimuth/defaults/main.yml [hunk 2]
```diff
@@ -85,6 +85,7 @@ azimuth_metrics_tenant_metrics_url_template: >-
 # The documentation URL
 # By default, we use the generic Azimuth docs
 azimuth_documentation_url: https://azimuth-cloud.github.io/azimuth-user-docs/
+azimuth_support_url: None
```
[the remark this answers is not in the document]
Comment (JohnGarbutt) [reply roles/azimuth/defaults/main.yml:88]: ah, doh, I missed that, good point.

[the remark this answers is not in the document]
Comment (thinkC) [reply roles/azimuth/defaults/main.yml]: Thanks @mkjpryor . Updated

**Review by mkjpryor — APPROVED** [review]
LGTM

**Pull request merged**